lib/services/readProvider.js

```diff
@@ -16,6 +16,7 @@ export const readProvider = async function (networkName) {
 
   try {
     if (networkName) {
+      // TODO: Update this to use the rpc values in networks.js
```

Comment:
We can add rpc urls for the ethereum networks as well, then this would be straightforward.

Infura needs api keys though. So I was hesitant to add that in case it injects our api key into peoples wallets when they click `Connect to Mainnet` in the notification banner.

Reply:
We could handle the Ethereum networks differently than the RPC-based ones. 👍